apps/transloco-playground/src/app/custom-transpilers/custom-transpilers.component.html

```diff
@@ -0,0 +1,17 @@
+<h2>Custom Transpilers</h2>
+<p>In order to test this, you will need to provide the <code>customTranspiler</code> in app.module.ts</p>
```

Comment:
Provide the custom transpiler in the app module, it's just logging the params now 🙂 
I don't think we need a dedicated page.

Reply:
Removed page